docs/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Stateful Functions - An Apache Flink® Project"
```

Comment:
Would it be better to rename this title to be, `Apache Flink Stateful Functions Documentation`, to be more consistent with how the Flink docs look?:

![image](https://user-images.githubusercontent.com/5284370/77506026-b7291600-6e9f-11ea-981b-df51d207fda7.png)

I also like how the Flink docs home page has a line providing the meta information of the docs build:
`This documentation is for Apache Flink version xxx. These pages were built at yyy`.